### DISCUSSION TITLE
Select next and prev item custom keymaps

### DISCUSSION THREAD

**Discussion by s1n7ax** [ibhagwan/fzf-lua#1320] [Q&A] [answered]
Colemak-dh layout user here. I would like to change the keybinds to select next and previous items in from the menu.

In telescope we can use this to change the keymaps. Is there something similar?

```
['<C-n>'] = 'move_selection_next',
['<C-e>'] = 'move_selection_previous',
``

**Comment by ibhagwan** [comment, marked as answer]
Use `kemap.fzf` to set the desired keys to up|down, search the issues how to do that

**Reply by s1n7ax** [reply]
```
[Fzf-lua] fzf error 2: unsupported key: <c-n>
```

```
return {
	'ibhagwan/fzf-lua',
	dependencies = { 'nvim-tree/nvim-web-devicons' },
	opts = {
		keymap = {
			builtin = {
				-- ['<c-n>'] = 'preview-page-down',
				-- ['<c-e>'] = 'preview-page-up',
			},
			fzf = {
				['<c-n>'] = 'up',
				['<c-e>'] = 'down',
			},
		},
	},
}

```

**Reply by ibhagwan** [reply]
Read man fzf, fzf binds are written as `ctrl-n`.

**Reply by CheeseTurtle** [reply]
Ok so why doesn't it accept `["alt-shift-n"]` as a key when it has `["alt-shift-up"]` and `["alt-shift-down"]` as keys in the default settings?

Also, I've noticed that `"top"` is an accepted action but `"bottom"` is reported as 'unknown', even though they appear to be defined right next to one another in the source.

**Reply by ibhagwan** [reply]
These aren’t valid binds in fzf, read `man fzf` for all acceptable binds.

**Comment by s1n7ax** [comment]
This works! Thanks

```
return {
	'ibhagwan/fzf-lua',
	dependencies = { 'nvim-tree/nvim-web-devicons' },
	opts = {
		fzf_bin = 'sk',
		keymap = {
			fzf = {
				['ctrl-e'] = 'up',
				['ctrl-n'] = 'down',
			},
		},
	},
}
```
